src/app/common/stores/applications/Distribution/Wallet.store.ts

```diff
@@ -169,9 +249,25 @@ class WalletStore extends BaseStore implements Wallet {
    * @param networkId: any
    */
   async networkHandler(networkId: any) {
-    console.log('networkId: ', networkId);
-    this.networkId = networkId;
-    this.wrongNetwork = networkId !== 1 && networkId !== undefined;
+    const notIncludeMainnet = NETWORKS_DATA.every((network: NetworkDataType) => {
+      return toHexString(network.networkId).toLowerCase() !== '0x1';
```

Comment:
This you can take from helpers, you can merge other branch when it's done to this branch